dask/utils.py

```diff
@@ -1056,6 +1056,8 @@ def ensure_dict(d: Mapping[K, V], *, copy: bool = False) -> Dict[K, V]:
 class OperatorMethodMixin(object):
     """A mixin for dynamically implementing operators"""
 
+    __slots__ = ()
```

Comment:
Why this change?

Reply:
So that Delayed will no longer have a ``__dict__``. A class will have a dict unless itself and all of its parent classes declare ``__slots__``. The change is inconsequential for other classes that derive from the same mixin that don't define ``__slots__``.